docs/src/main/asciidoc/datasource.adoc

```diff
@@ -445,22 +445,26 @@ AgroalDataSource inventoryDataSource;
 [[datasource-active]]
 === Activate or deactivate datasources
 
-When a datasource is configured at build time, it is active by default at runtime.
+When a datasource is configured at build time, it is active at runtime as soon as its URL is set.
```

Comment:
maybe?
```suggestion
When a datasource is configured at build time and its URL is set at runtime, it is active by default.
```
it feels like if we say `as soon as` it's more about the timing of activating the datasource rather than the fact itself, but IDK 🙈 🤷🏻‍♂️